src/shared-components/Card/ListItem/GenericListItems.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+} from '@mui/material';
+
+interface GenericListItemsProps {
+  title: string;
+  description?: string;
+  secondaryAction?: React.ReactNode;
+}
+
+export const GenericListItems = ({
```

Comment:
I think I made a mistake suggesting to keep the list component. After looking at the usage of list items on different pages, I think it's better to remove this component completely. Just keep individual list items.

Reply:
I removed GenericListItems page ı added lists individually.